Makefile

```diff
@@ -121,22 +109,29 @@ shell: build ## start a shell inside the build env
 	$(DOCKER_RUN_DOCKER) bash
 
 test: build ## run the unit, integration and docker-py tests
-	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary cross test-unit test-integration-cli test-docker-py
+	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary test-unit test-integration-cli test-docker-py
 
 test-docker-py: build ## run the docker-py tests
 	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary test-docker-py
 
 test-integration-cli: build ## run the integration tests
 	$(DOCKER_RUN_DOCKER) hack/make.sh build-integration-test-binary dynbinary test-integration-cli
 
-test-unit: build ## run the unit tests
-	$(DOCKER_RUN_DOCKER) hack/make.sh test-unit
+dobi:
+	curl -LsS -o dobi https://github.com/dnephin/dobi/releases/download/v0.8/dobi-linux
```

Comment:
Something like that should work (couldn't test, and I am not sure about how `chmod +x ..` works on windows.. 😅).

```diff
diff --git a/Makefile b/Makefile
index 035be0a..09eff52 100644
--- a/Makefile
+++ b/Makefile
@@ -73,6 +73,23 @@ ifeq ($(INTERACTIVE), 1)
 	DOCKER_FLAGS += -t
 endif
 
+dobicmd := "dobi"
+ifeq ($(OS),Windows_NT)
+	uname_S := Windows
+else
+	uname_S := $(shell uname -s)
+endif
+ifeq ($(uname_S), Windows)
+	dobicmd = "dobi.exe"
+	dobirelease = "https://github.com/dnephin/dobi/releases/download/v0.8/dobi-windows.exe"
+else ifeq ($(uname_S), Darwin)
+	dobirelease = "https://github.com/dnephin/dobi/releases/download/v0.8/dobi-darwin"
+else ifeq ($(uname_S), Linux)
+	dobirelease = "https://github.com/dnephin/dobi/releases/download/v0.8/dobi-linux"
+else
+	$(error unsupported platform)
+endif
+
 DOCKER_RUN_DOCKER := $(DOCKER_FLAGS) "$(DOCKER_IMAGE)"
 
 default: binary
@@ -125,21 +142,21 @@ test-docker-py: build ## run the docker-py tests
 test-integration-cli: build ## run the integration tests
 	$(DOCKER_RUN_DOCKER) hack/make.sh build-integration-test-binary dynbinary test-integration-cli
 
-dobi:
-	curl -LsS -o dobi https://github.com/dnephin/dobi/releases/download/v0.8/dobi-linux
-	chmod +x dobi
+$(dobicmd):
+	curl -LsS -o $(dobicmd) $(dobirelease)
+	chmod +x $(dobicmd)
 
-test-unit: dobi ## run the unit tests
-	./dobi test-unit
+test-unit: $(dobicmd) ## run the unit tests
+	./$(dobicmd) test-unit
 
 tgz: build cross ## build the archives (.zip on windows and .tgz\notherwise) containing the binaries
 	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary binary tgz
 
-validate: dobi ## run full validation
-	./dobi validate
+validate: $(dobicmd) ## run full validation
+	./$(dobicmd) validate
 
-cross: bundles dobi ## cross build the binaries for darwin, freebsd and\nwindows
-	./dobi cross
+cross: bundles $(dobicmd) ## cross build the binaries for darwin, freebsd and\nwindows
+	./$(dobicmd) cross
 
 win: build ## cross build the binary for windows
 	$(DOCKER_RUN_DOCKER) hack/make.sh win
```